overcast.py

```diff
@@ -29,7 +29,15 @@ def _get_html(self, url):
     def get_active_episodes(self):
         doc = self._get_html('https://overcast.fm/podcasts')
         return [
-            self.get_episode_detail(cell.attrib['href'])
+            # NOTE: If the hardcoded audio_type causes any problems, just uncomment the line below and comment out the dictionary below it.
+            # self.get_episode_detail(cell.attrib['href'])
+            {
+                'id': urlparse.urljoin('https://overcast.fm', cell.attrib['href']).lstrip('/'),
+                'title': cell.cssselect('div.titlestack div.title')[0].text_content(),
+                'audio_type': 'audio/mpeg',
+                'podcast_title': cell.cssselect('div.titlestack div.caption2')[0].text_content(),
+                'albumArtURI': cell.cssselect('img')[0].attrib['src'],
+            }
```

Comment:
This means we no longer need to make 2 requests per episode, and only once queued/played are the necessary calls made! In my testing, I have found that `'audio_type'` simply needs to be present valid, but can be different from the final played file. I tested setting it to `audio/aac` even though the actual file I tested with was `audio/mpeg` and it still worked. I will leave the note for future clarity in case something ever breaks.